agent/asm/mocks/secretsmanagerapi_mocks.go

```diff
@@ -137,6 +137,50 @@ func (mr *MockSecretsManagerAPIMockRecorder) CreateSecretWithContext(arg0, arg1
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "CreateSecretWithContext", reflect.TypeOf((*MockSecretsManagerAPI)(nil).CreateSecretWithContext), varargs...)
 }
 
+// DeleteResourcePolicy mocks base method
+func (m *MockSecretsManagerAPI) DeleteResourcePolicy(arg0 *secretsmanager.DeleteResourcePolicyInput) (*secretsmanager.DeleteResourcePolicyOutput, error) {
```

Comment:
I thought these unused methods' mocks were removed before by using an interface.

Reply:
this is what was generated from running make gogenerate, if we had some configuration to remove unused methods it would've not added back in?

Reply:
It may be that the generate command isn't correct -- @sharanyad, can we fix that in a seperate PR?

Reply:
got it. I thought this [comment](https://github.com/aws/amazon-ecs-agent/pull/1602#discussion_r223439039) was resolved. 
yeah, we can fix this later.